packages/@vuepress/markdown/lib/highlight.js

```diff
@@ -46,6 +64,16 @@ module.exports = (str, lang) => {
       logger.warn(chalk.yellow(`[vuepress] Syntax highlight for language "${lang}" is not supported.`))
     }
   }
+
+  const docLang = getDocLangCodeFromLang(lang)
+  if (docLang && !prism.languages[docLang]) {
```

Comment:
If we're doing this check here, wouldn't we also need to do the same check on line 77?

Reply:
Loading the corresponding doc-language before the language itself, but not all language has its doc-language, I think it's fine here